llvm/lib/Transforms/Yk/ControlPoint.cpp

```diff
@@ -86,24 +81,25 @@ const unsigned CPShadow = 13;
 
 using namespace llvm;
 
-/// Find the call to the dummy control point that we want to patch.
-/// Returns either a pointer the call instruction, or `nullptr` if the call
-/// could not be found.
-/// YKFIXME: For now assumes there's only one control point.
-CallInst *findControlPointCall(Module &M) {
+/// Find the calls to the dummy control point that we want to patch.
+/// Returns either a pointer the a vector of instructions
+std::vector<CallInst *> findControlPointCalls(Module &M) {
```

Comment:
Now we have two control points in both original and the cloned `main` functions.

`main` (stack map id: 0):
```
21:                                               ; preds = %18
  call void @__yk_trace_basicblock(i32 0, i32 9), !dbg !70
  %22 = load ptr, ptr %4, align 8, !dbg !70
  call void @llvm.dbg.value(metadata ptr %22, metadata !41, metadata !DIExpression()), !dbg !36
  call void (i64, i32, ptr, i32, ...) @llvm.experimental.patchpoint.void(i64 0, i32 13, ptr @__ykrt_control_point, i32 3, ptr %22, ptr %3, i64 0, ptr %3, ptr %4, ptr %5, ptr %6, ptr %22), !dbg !72
  br label %23, !dbg !73
```
`__yk_clone_main` (stack map id: 1): 
```
21:                                               ; preds = %18
  %22 = load ptr, ptr %4, align 8, !dbg !113
  call void @llvm.dbg.value(metadata ptr %22, metadata !93, metadata !DIExpression()), !dbg !89
  call void (i64, i32, ptr, i32, ...) @llvm.experimental.patchpoint.void(i64 1, i32 13, ptr @__ykrt_control_point, i32 3, ptr %22, ptr %3, i64 1, ptr %3, ptr %4, ptr %5, ptr %6, ptr %22), !dbg !115
  br label %23, !dbg !116

****
```